server/src/pages/settings/model.js

```diff
@@ -0,0 +1,25 @@
+const db = require("../../db");
+
+exports.createModel = function(req) {
+  let title = "Settings";
+  let model = {
+    title,
+    docTitle: title,
+  };
+  return db.select(`
+    SELECT accounts.avatarurl, accounts.nickname, accounts.email FROM accounts, devices
```

Comment:
This refers to `accounts.email` but that's not in the migration.  I'm guessing your database got out of sync with the migrations.